sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -2746,6 +2746,182 @@ abstract class JsonSuite
     }
   }
 
+  test("SPARK-37360: Write and infer TIMESTAMP_NTZ values with a non-default pattern") {
+    withTempPath { path =>
+      val exp = spark.sql("select timestamp_ntz'2020-12-12 12:12:12' as col0")
```

Comment:
It is a dataset/dataframe not an expression, just in case.

Reply:
It means "expected", not "expression".